src/engine.rs

```diff
@@ -294,25 +275,11 @@ pub fn run_treefmt(
     ret
 }
 
-fn print_summary(
-    traversed_files: usize,
-    matched_files: usize,
-    filtered_files: usize,
-    reformatted_files: usize,
-    start_time: std::time::Instant,
-) {
+fn print_summary(traversed_files: usize, reformatted_files: usize, start_time: std::time::Instant) {
     println!(
-        r#"
-traversed {} files
-matched {} files to formatters
-left with {} files after cache
-of whom {} files were re-formatted
-all of this in {:.0?}
```

Comment:
I should have said in the original issue; that info is useful to have in debug!() to better understand the perf profile.

Reply:
> I should have said in the original issue; that info is useful to have in debug!() to better understand the perf profile.

Ah, that makes more sense